scmrepo/fs.py

```diff
@@ -172,7 +172,7 @@ def __init__(
             scm = scm or Git(path)
             resolver = rev_resolver or Git.resolve_rev
             resolved = resolver(scm, rev or "HEAD")
-            tree_obj = scm.pygit2.get_tree_obj(rev=resolved)
```

Comment:
Hm, looks like we were always force-using pygit2 during our normal operation. This might have an unexpected impact...